src/components/Navbar.tsx

```diff
@@ -1,129 +1,161 @@
+              (open ? "-rotate-45 top-[40%] w-[120%]" : "w-full top-[80%]")} />
+          </button>
+        </div>
 
-                <div className={`
-                    flex justify-between flex-col items-stretch gap-8
-                    bg-darkblue lg:bg-transparent
-                    absolute w-full transition-all duration-300
-                    top-0 pt-20
+        <div className={`
+          flex justify-between flex-col items-stretch
+          bg-darkblue lg:bg-transparent
+          absolute w-full transition-all duration-300
+          top-0 pt-20
 
-                    lg:left-0 lg:flex-row lg:items-center lg:px-8 lg:pt-0
-                ` + (open ? "left-0" : "-left-full")
-                }>
-                    <a
-                        className="sr-only focus:not-sr-only"
-                        href="#main-content"
-                    >{nav.toContent}</a>
-                    <NavLink class="px-14 lg:px-0" href="/">{nav.home}</NavLink>
-                    <NavLink class="px-14 lg:px-0" href="/förföretag">{nav.forCompanies}</NavLink>
-                    {/*<NavLink class="px-14 lg:px-0" href="/förstudenter">{nav.forStudents}</NavLink>*/}
-                    {/*<NavLink class="px-14 lg:px-0" href="/mässan">{nav.about}</NavLink>*/}
-                    <div className="
-                        flex flex-row justify-center items-center
-                        py-4 gap-8
-                        lg:px-0 bg-blue lg:bg-transparent lg:ml-auto
-                    ">
-                        <Link
-                            className="bg-cerise py-2.5 px-4 rounded-full text-center hover:scale-105 transition-transform"
-                            href="/företagsanmälan"
-                        >{nav.companyForm}</Link>
-                        <button data-dont-close onClick={swapLocale} className={`
-                            w-8 h-8
-                            rounded-full border-none
-                            bg-no-repeat bg-center bg-[length:180%]
-                            transition-transform hover:scale-110
-                            ` + (locale == "sv"
-                                ? "bg-[url(/img/uk-flag.svg)]"
-                                : "bg-[url(/img/se-flag.svg)]"
-                            )}
-                        ><img className="sr-only" alt={nav.changeLanguage} /></button>
-                    </div>
-                </div>
-            </nav>
-        </>
-    );
+          lg:left-0 lg:flex-row lg:items-center lg:px-8 lg:pt-0
+        ` + (open ? "left-0" : "-left-full")
+        }>
+          <a
+            className="sr-only focus:not-sr-only"
+            href="#main-content"
+          >{t.toContent}</a>
+          <NavLink class="px-14 lg:px-4 p-4" href="/">{t.home}</NavLink>
+          <Group links={[
+            { href: "/förföretag", text: t.forCompanies },
+            { href: "/faq", text: "faq" },
```

Comment:
Dropdown doesn't isn't really responsive:

<img width="306" alt="image" src="https://user-images.githubusercontent.com/66337956/230063075-b36f6599-f66c-46a9-bae2-5eead822cc62.png">

Think something like lava does would work here:

![IMG_3604](https://user-images.githubusercontent.com/66337956/230063486-2c809b82-ba33-42b3-91b4-7e26fdfbc10d.PNG)

Reply:
I agree with this as well. Good to get this change published, and create a new PR for this afterwards.

Reply:
Sounds good 👍